include/seqan3/search/fm_index/bi_fm_index_cursor.hpp

```diff
@@ -33,7 +33,7 @@ namespace seqan3
  */
 
 /*!\brief The SeqAn Bidirectional FM Index Cursor.
- * \implements seqan3::bi_fm_index_cursor_specialisation
+ * \implements seqan3::detail::template_specialisation_of a seqan3::bi_fm_index_cursor.
```

Comment:
```suggestion
```

Since the concept does not exist anymore, the `seqan3::bi_fm_index_cursor` cannot implement it anymore. 
Saying it implements `seqan3::detail::template_specialisation_of a seqan3::bi_fm_index_cursor` is the same as saying an chicken looks like a chicken :chicken: :D

Reply:
Thanks for the feedback! I see what you mean by this wording being a bit nondescript :smiley: